### DISCUSSION TITLE
2022/11/28/Vitality

### DISCUSSION THREAD

**Discussion by giscus** [KMIT-Students/blog-demo#13] [General]
# 2022/11/28/Vitality

we really try to think that things are over when we encounter a disappointment and failure towards something terribly which went wrong our way and we ended up failing, but in an actual sense “what done is done when we say it’s done, that is it’s easy the world recommends us to try another time but how long we tend to waste a lot of time in picking up ourselves again, so the keynote for this is if we figure out a way where our reaction time or the time we take to pick ourselves up as short as possible that’s when we could do something remarkable because majority amount of our time we are putting it into investing in our selves and trying hard for that piercing breakthrough in our lives which is the only thing which could satisfy our hunger and self-esteem, ego obviously. Now the question comes to our mind that is how to decrease this amount of reaction time?

https://kmit-anubhav.github.io/2022/11/28/Vitality.html

**Comment by MaruthiCharan2403** [comment]
SUPER BRO FANTASTIC
